src/services/svc-service.ts

```diff
@@ -0,0 +1,51 @@
+import { CLIError } from '@oclif/errors'
+import * as fs from 'node:fs'
+import * as path from 'node:path'
+import { SVC_FOLDER } from '../paths'
+import { BundleDescriptor } from '../models/bundle-descriptor'
+import { BundleDescriptorService } from './bundle-descriptor-service'
+import { FSService } from './fs-service'
+
+export class SvcService {
+  private readonly parentDirectory: string
+  private readonly bundleDescriptorService: BundleDescriptorService
+  private readonly bundleDescriptor: BundleDescriptor
+  private readonly filesys: FSService
```

Comment:
fsService

Reply:
this is removed anyways since @zonia3000 found out it was unused